ContentPatcher/IManagedValue.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace ContentPatcher
+{
+    /// <summary>A set of parsed values linked to the Content Patcher context. These values are <strong>per-screen</strong>, so the result depends on the screen that's active when calling the members.</summary>
```

Comment:
We should update/add the code docs to refer to values instead of conditions (including in the other files too).